frontend/ui/widget/imageviewer.lua

```diff
@@ -479,13 +502,9 @@ function ImageViewer:onTap(_, ges)
             show_next_image = not BD.mirroredUILayout()
         end
         if show_prev_image then
-            if self._images_list_cur > 1 then
-                self:switchToImageNum(self._images_list_cur - 1)
-            end
+            self:onPageBack()
```

Comment:
Given that we are in an image(s) context (and not book/pages), best to name your new functions:
```lua
onShowPrevImage()
onShowNextImage()
```